assets/xml/objects/object_kitan.xml

```diff
@@ -29,26 +29,26 @@
-        <Limb Name="object_kitan_Standardlimb_007F40" Type="Standard" EnumName="OBJECT_KITAN_LIMB_13" Offset="0x7F40" />
-        <Limb Name="object_kitan_Standardlimb_007F4C" Type="Standard" EnumName="OBJECT_KITAN_LIMB_14" Offset="0x7F4C" />
-        <Skeleton Name="object_kitan_Skel_007FA8" Type="Flex" LimbType="Standard" LimbNone="OBJECT_KITAN_LIMB_NONE" LimbMax="OBJECT_KITAN_LIMB_MAX" EnumName="ObjectKitanLimb" Offset="0x7FA8" />
+        <Limb Name="object_kitan_Standardlimb_007E68" Type="Standard" EnumName="KEATON_LIMB_LOWER_BODY" Offset="0x7E68" />
+        <Limb Name="object_kitan_Standardlimb_007E74" Type="Standard" EnumName="KEATON_LIMB_UPPER_BODY" Offset="0x7E74" />
+        <Limb Name="object_kitan_Standardlimb_007E80" Type="Standard" EnumName="KEATON_LIMB_HEAD" Offset="0x7E80" />
+        <Limb Name="object_kitan_Standardlimb_007E8C" Type="Standard" EnumName="KEATON_LIMB_LEFT_SHOULDER" Offset="0x7E8C" />
+        <Limb Name="object_kitan_Standardlimb_007E98" Type="Standard" EnumName="KEATON_LIMB_LEFT_ARM" Offset="0x7E98" />
+        <Limb Name="object_kitan_Standardlimb_007EA4" Type="Standard" EnumName="KEATON_LIMB_RIGHT_SHOULDER" Offset="0x7EA4" />
+        <Limb Name="object_kitan_Standardlimb_007EB0" Type="Standard" EnumName="KEATON_LIMB_RIGHT_ARM" Offset="0x7EB0" />
+        <Limb Name="object_kitan_Standardlimb_007EBC" Type="Standard" EnumName="KEATON_LIMB_RIGHT_LEG" Offset="0x7EBC" />
+        <Limb Name="object_kitan_Standardlimb_007EC8" Type="Standard" EnumName="KEATON_LIMB_RIGHT_FOOT" Offset="0x7EC8" />
+        <Limb Name="object_kitan_Standardlimb_007ED4" Type="Standard" EnumName="KEATON_LIMB_LEFT_LEG" Offset="0x7ED4" />
+        <Limb Name="object_kitan_Standardlimb_007EE0" Type="Standard" EnumName="KEATON_LIMB_LEFT_FOOT" Offset="0x7EE0" />
+        <Limb Name="object_kitan_Standardlimb_007EEC" Type="Standard" EnumName="KEATON_LIMB_RIGHT_TAIL_BASE" Offset="0x7EEC" />
+        <Limb Name="object_kitan_Standardlimb_007EF8" Type="Standard" EnumName="KEATON_LIMB_RIGHT_TAIL_MID" Offset="0x7EF8" />
+        <Limb Name="object_kitan_Standardlimb_007F04" Type="Standard" EnumName="KEATON_LIMB_RIGHT_TAIL_END" Offset="0x7F04" />
+        <Limb Name="object_kitan_Standardlimb_007F10" Type="Standard" EnumName="KEATON_LIMB_MIDDLE_TAIL_BASE" Offset="0x7F10" />
+        <Limb Name="object_kitan_Standardlimb_007F1C" Type="Standard" EnumName="KEATON_LIMB_MIDDLE_TAIL_MID" Offset="0x7F1C" />
+        <Limb Name="object_kitan_Standardlimb_007F28" Type="Standard" EnumName="KEATON_LIMB_MIDDLE_TAIL_END" Offset="0x7F28" />
+        <Limb Name="object_kitan_Standardlimb_007F34" Type="Standard" EnumName="KEATON_LIMB_LEFT_TAIL_BASE" Offset="0x7F34" />
+        <Limb Name="object_kitan_Standardlimb_007F40" Type="Standard" EnumName="KEATON_LIMB_LEFT_TAIL_MID" Offset="0x7F40" />
+        <Limb Name="object_kitan_Standardlimb_007F4C" Type="Standard" EnumName="KEATON_LIMB_LEFT_TAIL_END" Offset="0x7F4C" />
```

Comment:
Any reason not to just name these limbs? I took a look and it looks like the `EnumName`s are all right; seems like we could probably just name the limbs appropriately too and knock these out.